justfile

```diff
@@ -41,18 +41,26 @@ test-wkspc:
     cargo test --workspace -- --nocapture
 
 # Execute tests/prepare.sh.
-test-functional-prepare:
-    bash tests/prepare.sh
+test-functional-prepare arg="":
+    bash tests/prepare.sh {{arg}}
 
 # Execute tests/run.sh
-test-functional-run:
-    bash tests/run.sh
-
-# Execute our python integration tests inside /tests.
-#
-# Make sure you have done the necessary setup explained in our README.md in the root of the folder.
-test-functional-uv-run:
-    uv run tests/run_tests.py
+test-functional-run arg="":
+    bash tests/run.sh {{arg}}
+
+# format and lint functional tests 
+test-functional-uv-fmt:
+    uv run black --check --verbose ./tests
+    uv run pylint --verbose ./tests
+
+# Format functional tests
+# Lint functional tests
+# Prepare functional tests
+# Run functional tests
+test-functional:
```

Comment:
Collapse those into one line, or just will only show the last one in the help list